app/javascript/controllers/map/tokens_controller.js

```diff
@@ -139,6 +159,9 @@ export default class extends Controller {
     const tokenId = event.dataTransfer.getData("text/plain")
 
     const token = this.findToken(tokenId)
+    if (this.hasDrawerTarget) {
+      this.drawerTarget.dispatchEvent(new CustomEvent('close'))
```

Comment:
If I follow this correctly, the token drawer will close every time a token is dropped on the map?

Let's see how that feels in practice as it might be a pain for adding multiple creatures

Reply:
That's right, ultimately I thought it should stay open for a bit and then close after some time of not interacting with it, so I decided to do this for now and then improve later.

I also included in my work on token selection, for which a PR is upcoming soon, a keyboard shortcut to open and close the token drawer.

Reply:
Happy to see how this works out. Keyboard shortcuts will be grand 👍 and the only other way I was thinking was clicking on the map could close any open drawers